docs/pages/guides/store-assets.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: Create Store Assets
+description: Learn how to create screenshots and previews for your app's store pages.
+sidebar_title: Store assets
+---
+
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+
+Before you can submit your app to the App Store and Play Store, you need to provide some assets for your store listing page.
+The goal of these images and videos is to give prospective users an idea of what your app experience
+is going to be like.
+
+The most important assets you need to add are the app screenshots. Note that while they are called "app screenshots" on both stores, and they *do* have to include accurate
```

Comment:
do we have anything to support that these are the most important? if not maybe we can leave it out

Reply:
They're the only one needed for both stores. I'll say that instead.